pkg/operator/ceph/object/objectstore.go

```diff
@@ -358,3 +361,73 @@ func GetObjectBucketProvisioner(c *clusterd.Context, namespace string) string {
 	}
 	return provName
 }
+
+func checkDashboardUser(context *Context) bool {
```

Comment:
What does this function do? Add a little comment here to explain.

Reply:
Done